admin_manual/configuration_server/occ_command.rst

```diff
@@ -276,13 +294,61 @@ This is example converts to SQLite MySQL/MariaDB::
 For a more detailed explanation see 
 :doc:`../configuration_database/db_conversion`
 
+.. _encryption_label:
+
 Encryption
 ----------
 
-When you are using encryption, you must manually migrate your encryption 
-keys after upgrading your ownCloud server::
+ownCloud 8.2 introduces a new set of encryption commands::
+
+ encryption:change-key-storage-root  
+ encryption:show-key-storage-root    
+ encryption:set-default-module       
+ encryption:list-modules             
+ encryption:encrypt-all              
+ encryption:decrypt-all              
+ encryption:disable                  
+ encryption:status                   
+ encryption:enable
+ encryption:enable-master-key 
+ encryption:migrate
+
+``encryption:status`` shows whether you have active encryption, and your default encryption module. To enable encryption you must first enable the Encryption app, and then run ``encryption:enable``::
+
+ $ sudo -u www-data php occ app:enable encryption
+ $ sudo -u www-data php occ encryption:enable
+ $ sudo -u www-data php occ encryption:status
+  - enabled: true
+  - defaultModule: OC_DEFAULT_MODULE
+   
+``encryption:change-key-storage-root`` is for moving your encryption keys to a different folder. It takes one argument, ``newRoot``, which defines your new root folder::
+
+ $ sudo -u www-data php occ encryption:change-key-storage-root /etc/oc-keys
```

Comment:
probably a bad example. The new root must be somewhere in the ownCloud data folder. Otherwise all the internal OC Filesystem operations don't work. But of course you could mount/link "/etc/oc-keys" to "data/oc-key"  for example.